README.md

```diff
@@ -1,6 +1,13 @@
 # [Water Monitoring System](https://vinitshahdeo.github.io/Water-Monitoring-System/) - IoT Project
+---
+> ## :mask: [COVID-19](http://corona-cases-india.netlify.com/) | Stay Home, Stay Safe! :house:  <br> :mag_right: Checkout [this](http://corona-cases-india.netlify.com/) web app for live CORONA updates. 
```

Comment:
These emojis won't work in `gh-pages`. Please remove it.

Reply:
Added jemoji plugin in the [_config.yml](https://github.com/vinitshahdeo/Water-Monitoring-System/blob/gh-pages/_config.yml) file.
Tested locally and it worked.